--- LineageLayoutView.js
+++ LineageLayoutView.js
@@ -431,9 +431,7 @@ define(['require',
                     status: 'status',
                     classificationNames: 'classifications',
                     meanings: 'term' ,
-                    data : 'data' ,
-                    sla_flag : 'sla_flag',
-                    sla_minutes : 'sla_minutes'
+                    data : 'data'
                 };
                 var data = {};
                 _.each(config, function(valKey, key) {
@@ -443,13 +441,19 @@ define(['require',
                     }
                     if (val) {
                         if(key.match('data')) {
-                            data['starttime']=new Date(parseInt(val[0]['attributes']['starttime'])).toGMTString()
-                            data['endtime']=new Date(parseInt(val[0]['attributes']['endtime'])).toGMTString()
+                            val.sort(function (a, b) {
+                                    if(b['attributes']['endtime'] >  a['attributes']['endtime']){return 1;}
+                                    if(b['attributes']['endtime'] <  a['attributes']['endtime']){return -1;}
+                                    return 0;
+                                    }
+                            )
+                            data['starttime']=new Date(parseInt(val[0]['attributes']['starttime'])).toGMTString();
+                            data['endtime']=new Date(parseInt(val[0]['attributes']['endtime'])).toGMTString();
                         }
                         else {
-                        data[valKey] = val;
+                            data[valKey] = val;
                         }
-                    }
+                   }
                 });
                 this.ui.nodeDetailTable.html(CommonViewFunction.propertyTable({
                     "scope": this,
